_pages/ato/checklist.md

```diff
@@ -3,7 +3,9 @@ title: ATO Checklist
 navtitle: Checklist
 ---
 
-To start the security clearance process, [create an issue in the Infrastructure repository](https://github.com/18F/Infrastructure/issues/new?title=ATO+for+%3Cproject%3E+-+due+%3Cdeadline%3E) using [this template](https://raw.githubusercontent.com/18F/before-you-ship/master/_includes/checklist.md). Make sure to replace the placeholders at the top. Feel free to add a username after each task to assign it, and/or make corresponding items in your issue tracker. After the Infrastructure issue is created, the Infrastructure Team will schedule a time to meet with you and discuss the ATO.
+To start the security clearance process, [create an issue in the Infrastructure repository](https://github.com/18F/Infrastructure/issues/new?title=ATO+for+%5Bproject%5D-+due+%5Bdeadline%5D) using [this template](https://raw.githubusercontent.com/18F/before-you-ship/master/_includes/checklist.md) as the body. Make sure to replace the placeholders (the things in `[square brackets]`). Feel free to add a username after each task to assign it, and/or make corresponding items in your issue tracker. Unless otherwise specified, all tasks are the responsibility of the project team.
```

Comment:
This page appears in Before You Ship, but not in the checklist template (and thus, not the ATO issue that will be created).